Sources/MongoKitten/Commands/AggregateCommand.swift

```diff
@@ -225,7 +227,7 @@ extension AggregateCursor where Element == Document {
             .getFirstResult()
             .thenThrowing { result in
                 guard let result = result else {
-                    throw MongoKittenError(.unexpectedNil, reason: .noResultDocument)
+                    return 0
```

Comment:
Why did you change this? Is there a case where MongoDB does not return a document here?

Reply:
No. MongoDB only returns a Document is the pipeline input >= 1 document. Which is ridiculous